vignettes/02-get-weather-data-R.Rmd

```diff
@@ -0,0 +1 @@
+# Weather Vignette
```

Comment:
I created this file in #85, so you might want to delete this file before merging this PR @kimberlyh66

Reply:
okay